atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -334,10 +333,7 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
             if arch in arches:
                 self.multilib_arches.append(arch)
         if self.multilib_arches:
-            if data.get('multilib_method'):
-                self.multilib_method = data.get('multilib_method')
-            else:
-                self.multilib_method = MULTILIB_METHOD_DEFAULT
+            self.multilib_method = data.get('multilib_method', [])
```

Comment:
Nit: could even be `data.get('multilib_method')` without providing an if-not-there value, because `None` is falsy just like `[]` is. In other words, the `multilib_method or MULTILIB_METHOD_DEFAULT` expression will evaluate to the same as `MULTILIB_METHOD_DEFAULT` regardless of which falsy value we set `self.multilib_method` to here.

Reply:
Need the [] to avoid triggering an error on sorted() later on, and need sorted() to keep the list ordering consistent between python2 and python3 test cases.

Reply:
Can't you use sorted() on both sides of the expression in the test, e.g. `assert sorted(body_json.get('multilib_method', [])) == sorted(expected_method)`?

Reply:
The way it's currently written, `[]` will get replaced with `MULTILIB_METHOD_DEFAULT` anyways in `odcs_utils.py`. Using None should be fine here, assuming, of course, that we remove the sorted calls later on in this file as mentioned in my previous comment.

There's a test in `test_resolve_plugins` that will fail because it expects `multilib_method` to be set to `[]` when it's `None`. IMO this is invalid at this layer. Applying a valid default value is done in `odcs_utils.py` and it should be tested in `test_odcs_utils.py`.

Reply:
Fixed.